Source/Urho3D/LuaScript/LuaScriptInstance.cpp

```diff
@@ -586,6 +596,8 @@ void LuaScriptInstance::GetScriptAttributes()
                     info.type_ = VAR_INTRECT;
                 else if (typeName == "Intvector2")
                     info.type_ = VAR_INTVECTOR2;
+                else if (typeName == "Intvector3")

```

Comment:
Looks like a bug to me. The string "Intvector2" cannot be found anywhere else in the code. Also other Lua stuff that deal with user types use "IntVector2". I'll fix it.